nixos/modules/services/x11/xserver.nix

```diff
@@ -512,7 +512,7 @@ in
             XKB_BINDIR = "${xorg.xkbcomp}/bin"; # Needed for the Xkb extension.
             XORG_DRI_DRIVER_PATH = "/run/opengl-driver/lib/dri"; # !!! Depends on the driver selected at runtime.
             LD_LIBRARY_PATH = concatStringsSep ":" (
-              [ "${xorg.libX11.out}/lib" "${xorg.libXext.out}/lib" ]
+              [ "${xorg.libX11.out}/lib" "${xorg.libXext.out}/lib" "/run/opengl-driver/lib" ]
```

Comment:
Oh, I assume by setting `LD_LIBRARY_PATH` here we overrode the system-wide setting. (Or it's possible it isn't propagated to services by default &ndash; I don't know.)